server/controllers/finance/reports/budget_analytical/index.js

```diff
@@ -135,6 +135,9 @@ async function report(req, res, next) {
         [BUDGET_ANALYSIS_REFERENCE_TYPE_ID],
       );
 
+      const accountNumbersByReference = accountsReferenceType[2];
```

Comment:
This can be done later, but it would be best if these numbers (eg, `[2]`) could be constants from bhConstants (eg, accountReference...).

Reply:
I just did it – thanks again for the feedback!